src/docs/content/builders/dialog.md

```diff
@@ -63,9 +74,16 @@ By default, we set the `role` attribute to `dialog`. If you want it to be consid
 dialog, you can set the `role` builder prop to `alertdialog`.
 
 ```ts {2}
-const { trigger, portal, overlay, content } = createDialog({
-	role: 'alertdialog'
-})
+const { 
+	elements: {
+		trigger, 
+		portal, 
+		overlay, 
+		content,
+		},
+	} = createDialog({
+	role: 'alertdialog',
+});
```

Comment:
The elements are not the most important part here, so I think we can omit them here. An example:
```suggestion
const { /* ... */ } = createDialog({
	role: 'alertdialog',
});
```

Also, the role should be highlighted. See the `\`\`\`ts {2}` part? The number indicates what line will be highlighted.

You can do the same for the rest of the snippets.